app/src/main/java/com/pileproject/drive/preferences/BlockPreferencesSchema.java

```diff
@@ -20,15 +20,13 @@
 
 /**
  * Schema for "block_preferences" table
- *
- * @author <a href="mailto:tatsuyaw0c@gmail.com">Tatsuya Iwanari</a>
- * @version 1.0 21-April-2016
  */
 @Table(name = "block_preferences")
 public class BlockPreferencesSchema {
```

Comment:
Oh, I should have added more detailed explanations, sorry :bow:

There are several things I should tell you.

1. We're using `SharedPreferences` (see: [基本的なSharedPreferencesの使い方 - Qiita]( https://goo.gl/K9jz6T )) to store `Preference`.

2. We're using `kvs-schema` plugin (see: [中規模以上のアプリでSharedPreferencesをエレガントに扱う - Qiita]( https://goo.gl/t0V0WF )) to use `SharedPreferences` more elegantly.

3. `HogeSchema` is used to make an auto-generated class whose name is `Hoge` (in this case, `BlockPreferences`) with the plugin.  

4. `kvs-schema` can set the default values in `Schema` like this class. But if we keep it in `main` build variant (thus, this is used over all variants), the schema forces all apps to use the same default values for sensors in this case. But I think they may be different from robot to robot (I'm not so sure about this).

5. I want to use `BlockPreferencesSchema` in each variant independently so that they can have different defaults and this also allows us to remove useless default values for not implemented sensors (e.g., `ColorSensor` for PILE robot).

This is the background and I'm wondering what the best way is :confused: